### PR TITLE
brie: support batch ddl for sql restore

### LINKED ISSUE #48301: br: support batch ddl for sql restore
## Enhancement
Restore sql execution does not support batch ddl as in restore clp. It's a performance issue, and we need to provide the support to SQL restore.

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: close #48301 

Problem Summary:
brie restore need batch ddl to accelerate the table creation。
### What changed and how does it work?
implement CreateTables function for brieGlue
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/executor/brie.go [hunk 1]
```diff
@@ -101,7 +100,7 @@ func (p *brieTaskProgress) Close() {
 	p.lock.Lock()
 	current := atomic.LoadInt64(&p.current)
 	if current < p.total {
-		p.cmd = fmt.Sprintf("%s Cacneled", p.cmd)
+		p.cmd = fmt.Sprintf("%s Cancelled", p.cmd)
```
Comment (Leavrth) [inline pkg/executor/brie.go:103]: `Cancelled` or `Canceled`?

Comment (YuJuncen) [reply pkg/executor/brie.go:103]: If I recall right TiKV prefers AmE (canceled). For me I think both will be OK.

File: tests/realtikvtest/brietest/backup_restore_test.go [hunk 2]
```diff
@@ -70,3 +71,32 @@ func TestBackupAndRestore(t *testing.T) {
...
+	tk.MustQuery("backup database br to 'local://" + tmpDir + "'")
+
+	// remove database for recovery
+	tk.MustExec("drop database br")
+
+	// restore database with backup data
+	tk.MustQuery("restore database * from 'local://" + tmpDir + "'")
+	tk.MustExec("use br")
+	for i := 0; i < tableNum; i += 1 {
+		tk.MustQuery(fmt.Sprintf("select count(*) from table_%d", i)).Check(testkit.Rows("1"))
```
Comment (YuJuncen) [inline tests/realtikvtest/brietest/backup_restore_test.go:99]: Maybe also query the DDL job history for making sure we have used batch create table.

**Review by YuJuncen — APPROVED** [review]
rest lgtm

**Review by 3pointer — APPROVED** [review]

**Pull request merged**